apps/java/services/storage-api/application.yml

```diff
@@ -19,7 +19,7 @@ keycloak.url: ${KEYCLOAK_URL:http://localhost:8082}
 
 # minio
 driver.storage.factoryclass: minio
-driver.storage.endpoint.url: ${MINIO_HOST:localhost}
+driver.storage.endpoint.url: ${MINIO_HOST:minio}
```

Comment:
I would keep 'localhost'. That would be the default for local development environment(no docker-compose)

Reply:
Agreed, reverted that change on both variables.